gitpod-io/gitpod
# [ws-manager] config terminationGracePeriodSeconds

## Description
Allow extending the termination grace period of workspace pods through gitpod.yml configuration.

## Related Issue(s)

Fixes #

## How to test

- Start a workspace on this branch : https://github.com/svenefftinge/termination-test/tree/se-extended-race
- run `ignoreSigterm > out.txt` 
- stop workspace. It should stay in stopping close to 100 seconds.
- when starting again, the `out.txt` should contain log messages that show the process ran almost 100 seconds after receiving the SIGHUP

## Release Notes

```release-note
Allow configuration of 'terminationGracePeriodSeconds' to get more time for gracefully stopping services in workspaces
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`
